libs/datamodel/core/tests/reformat/reformat.rs

```diff
@@ -293,6 +293,33 @@ model a {
     assert_reformat(input, expected);
 }
 
+#[test]
+fn reformatting_must_add_backrelation() {
```

Comment:
we have a specific test file for implicit relations. It would be better to put the test there: `reformat_implicit_relations.rs`